dist/pretender.js

```diff
@@ -0,0 +1,435 @@
+//TODO: remove from source control when the tests begin using the source file instead of this one.
```

Comment:
should this file committed?

Reply:
I thought we did, because it replaces `./pretender.js`, but we don't. `test-ci` triggers a build, duh!